README.md

```diff
@@ -37,6 +37,10 @@ To build locally with Drone:
 - Install the [Drone command line tools][dronecli].
 - Run `drone exec` from within the root directory of the govmomi repository.
 
+## Installation
+
+`go get github.com/vmware/govmomi`
+
```

Comment:
Thanks, can you add the `-u` flag and use the same markup as: https://github.com/vmware/govmomi/tree/master/govc#source

Reply:
Thanks review!